src/components/Form.tsx

```diff
@@ -84,3 +101,31 @@ export const FormHint: FunctionComponent<FormHintProps> = ({
     {children}
   </div>
 );
+
+export const Input: FunctionComponent<InputProps> = ({
+  name,
+  type = "text",
+  spellCheck = true,
+}: InputProps): JSX.Element => (
+  <input
+    className="govuk-input"
+    name={name}
+    type={type}
+    spellCheck={spellCheck}
+  />
+);
```

Comment:
![image](https://user-images.githubusercontent.com/54271433/107084766-6092a500-67ef-11eb-9de6-6a8a35132328.png)

Reply:
Missing `id` was intentional as I thought its only use was in client-side DOM manipulation and in the spirit of minimal necessary `props`, but your comment below on `<Label>` reminded me otherwise!  Adding `id` to `props` now.

Reply:
`id` added to `Input`